spec/factories/approvals.rb

```diff
@@ -1,10 +1,10 @@
 FactoryGirl.define do
```

Comment:
Bearing that in mind, I'm going to delete `spec/factories/steps.rb` for the moment.